### PR TITLE
49 rails user auth

### LINKED ISSUE #49: rails user auth

### PR DESCRIPTION
**Pull request by olmesm**
closes #49

### CODE REVIEW

File: app/controllers/crawls_controller.rb [hunk 1]
```diff
@@ -3,6 +3,9 @@ class CrawlsController < ApplicationController
   skip_before_action :verify_authenticity_token
 
   def index
+    p 'index'
```
Comment (hawksdoves) [inline app/controllers/crawls_controller.rb]: delete p

File: app/controllers/crawls_controller.rb [hunk 2]
```diff
@@ -17,6 +20,8 @@ def new
   end
 
   def create
+    p 'create'
```
Comment (hawksdoves) [inline app/controllers/crawls_controller.rb]: delete p

File: db/migrate/20160510102317_devise_token_auth_create_users.rb [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+      ## Tokens
+      t.json :tokens
+
+      t.timestamps
+    end
+
+    add_index :users, :email
+    add_index :users, [:uid, :provider],     :unique => true
+    add_index :users, :reset_password_token, :unique => true
+    # add_index :users, :confirmation_token,   :unique => true
```
Comment (hawksdoves) [inline db/migrate/20160510102317_devise_token_auth_create_users.rb]: why are these hashed out?

Comment (olmesm) [inline db/migrate/20160510102317_devise_token_auth_create_users.rb]: think its standard gen code

**Pull request merged**